ValheimPlus/Hud.cs

```diff
@@ -62,7 +62,16 @@ private static void Postfix(Skills __instance, Skills.SkillType skillType, float
 					__instance.m_player.Message(MessageHud.MessageType.TopLeft, skill.m_info.m_skill + " [" + Helper.tFloat(skill.m_accumulator, 2) + "/" + Helper.tFloat(skill.GetNextLevelRequirement(), 2) + "] (" + Helper.tFloat(percent, 0) + "%)", 0, skill.m_info.m_icon);
 				}
 			}
+		}
 
+		[HarmonyPatch(typeof(Hud), "DamageFlash")]
+		public static class RemoveDamageFlash
+		{
+			private static void Prefix(Hud __instance)
+			{
+				if (Configuration.Current.Hud.IsEnabled && Configuration.Current.Hud.removeDamageFlash)
+					__instance.m_damageScreen.enabled = false;
```

Comment:
.enabled is the equivalent of .SetActive(bool newvalue)

the normal code, launches .SetActive(true) rendering this .enabled moot

Reply:
I've changed it to postfix and updated the code to use the SetActive method to set said flag to false. Thank you for the suggestions! Change is in [this commit](https://github.com/valheimPlus/ValheimPlus/pull/141/commits/3936b745cb6d2f232a1ee86ac722051e19b1d0f0)